src/polyfills/document-contains.js

```diff
@@ -36,8 +36,9 @@ function documentContainsPolyfill(node) {
  * @param {!Window} win
  */
 export function install(win) {
-  if (!win.HTMLDocument.prototype.contains) {
-    win.Object.defineProperty(win.HTMLDocument.prototype, 'contains', {
+  const documentClass = win.HTMLDocument || win.Document;
```

Comment:
Could you please add a comment above this indicating for any future readers the context?